tests/virtualization/universal/patch_guests.pm

```diff
@@ -32,9 +32,10 @@ sub run {
     assert_script_run qq(echo -e "Guests before and after patching:" > $kernel_log);
     assert_script_run qq(echo -e "\\n\\nBefore:" >> $kernel_log);
     foreach my $guest (keys %virt_autotest::common::guests) {
-        ($guest_running_version, $guest_running_sp) = get_os_release("ssh root\@$guest");
+        record_info "$guest", "Probing the guest, adding test repositories and patching the system";
+        ensure_online $guest, ping_delay => 45 foreach (keys %virt_autotest::common::guests);
```

Comment:
magic number, why 45? Better define a variable with a good descriptive name and make 45s the default

Reply:
The `ensure_online()` already has default value. I'll try it.